Introduction to Kotlin Multiplatform/06_jvm_render.md

```diff
@@ -69,27 +78,35 @@ There should be several errors in the `colour.kt` file. We miss the
 declarations for colors in the `src/commonMain/kotlin/common.kt`:
 
 ```kotlin
-expect class Color
-
 object Colors
+
+expect class Color
 expect fun Colors.newColor(r: Int, g: Int, b: Int): Color
 expect val Colors.BLACK : Color
 ```
 
+The `expect class`, `expect fun`, and `expect val` declarations in the Kotlin/Common
+code instructs the compiler that we'd like to use these declarations in our Kotlin/Common
+code, but we are not able to provide implementations. Implementations are provided via the
+`actual` keyword independently for the JVM and JS targets (in general,
+[Kotlin/Native](https://kotlinlang.org/docs/reference/native-overview.html) can
+also be [used](https://kotlinlang.org/docs/tutorials/native/mpp-ios-android.html)
+to target iOS and other platforms).   
+
 Now we fix the code in the `colour.kt` file to use our expected `Color` class
```

Comment:
Now that we've fixed the code in the `colour.kt` file to use our expected `Color` class
and the `Colors` object to deal with colors (instead of using the `java.awt.Color` class).
We need to remove the import in the `colour.kt` and use the `Colors.newColor`
instead of the `Color` constructor call.


The last fix has to be performed in the `render.kt` file. We need to remove the
import of the `java.awt.Color` class and switch to the `Colors.BLACK`
property in the middle of the file.